app/src/main/java/pl/futurecollars/invoicing/utils/FileService.java

```diff
@@ -2,15 +2,24 @@
 
 import java.io.File;
 import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
 import java.util.List;
+import lombok.extern.slf4j.Slf4j;
 import org.apache.commons.io.FileUtils;
 
+@Slf4j
 public class FileService {
 
     private final File file;
 
     public FileService(String fileName) {
         file = new File(fileName);
+        try {
+            Files.createFile(Paths.get(fileName));
```

Comment:
Actually, if we have problem with creating a file this service should not be initialized, so it is better to throw some implementation of RuntimeException (or even checked Exception) than try catch in constructor.